cnames_active.js

```diff
@@ -1199,7 +1199,7 @@ var cnames_active = {
   "james": "jamesrowen.github.io/james", // noCF? (don´t add this in a new PR)
   "jameshrx": "sawyerbx.github.io/james",
   "jargon": "hugogiraudel.github.io/SJSJ", // noCF? (don´t add this in a new PR)
-  "jasonliang": "jasonliang-dev.github.io",
+  "jasonliang": "jasonliang-deCNAMEv.github.io",
```

Comment:
```suggestion
  "jasonliang": "jasonliang-dev.github.io",
```
Please revert this change.